tools/meson.build

```diff
@@ -238,6 +238,12 @@ tools = {
             'kvs_helper.c',
             'parm_groups.c',
         ),
+        'c_args': [
+            '-DHDR_HISTOGRAM_C_FROM_SUBPROJECT=@0@'.format(get_variable('HdrHistogram_c_from_subproject', 0)),
```

Comment:
I like this solution, but I think the variable is defined all the time.

I need to come up with a better solution to this. It sucks :). Maybe I will just write a small Meson build system for HdrHistogram_c.